app/controllers/application_controller.rb

```diff
@@ -3,9 +3,17 @@ class ApplicationController < ActionController::Base
   protect_from_forgery with: :exception
   helper_method :current_person
 
+  rescue_from Pundit::NotAuthorizedError, with: :user_not_authorized
```

Comment:
https://makandracards.com/makandra/41790-how-rails-chooses-error-pages-404-500-for-exceptions
https://makandracards.com/makandra/12807-custom-error-pages-in-rails-3-2+
http://guides.rubyonrails.org/layouts_and_rendering.html#the-status-option